src/main/java/com/github/wovnio/wovnjava/HtmlConverter.java

```diff
@@ -162,8 +162,9 @@ private void appendHrefLang() {
     }
 
     private void replaceNodeToMarkerComment(Element element) {
-        Comment comment = new Comment(WOVN_MARKER_PREFIX + String.valueOf(markers.size()));
-        element.replaceWith(comment);
+        String commentData = WOVN_MARKER_PREFIX + String.valueOf(markers.size());
+        element.before("<!--" + commentData + "-->");
+        element.remove();
```

Comment:
Nitpick: Because we no longer use the Comment class,  think we can remove its import also. (Line 9)